<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteQueueWidget</class>
 <widget class="QWidget" name="RemoteQueueWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox1">
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Queue configuration</string>
        </property>
        <layout class="QFormLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <property name="labelAlignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label1">
           <property name="text">
            <string>Job &amp;submission command:</string>
           </property>
           <property name="buddy">
            <cstring>edit_submissionCommand</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="edit_submissionCommand">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label2">
           <property name="text">
            <string>Request &amp;queue command:</string>
           </property>
           <property name="buddy">
            <cstring>edit_requestQueueCommand</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="edit_requestQueueCommand"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label3">
           <property name="text">
            <string>&amp;Launch script name:</string>
           </property>
           <property name="buddy">
            <cstring>edit_launchScriptName</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="edit_launchScriptName"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label4">
           <property name="text">
            <string>Remote &amp;working directory:</string>
           </property>
           <property name="buddy">
            <cstring>edit_workingDirectoryBase</cstring>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="edit_workingDirectoryBase"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label5">
           <property name="text">
            <string>Job &amp;cancel command:</string>
           </property>
           <property name="buddy">
            <cstring>edit_killCommand</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="edit_killCommand"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Default &amp;walltime limit:</string>
           </property>
           <property name="buddy">
            <cstring>wallTimeHours</cstring>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QSpinBox" name="wallTimeHours">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="suffix">
              <string> h</string>
             </property>
             <property name="maximum">
              <number>2000</number>
             </property>
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="wallTimeMinutes">
             <property name="suffix">
              <string> m</string>
             </property>
             <property name="maximum">
              <number>59</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label6">
           <property name="text">
            <string>Queue update interval:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="updateIntervalSpin">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="suffix">
            <string> m</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>120</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>SSH configuration</string>
        </property>
        <layout class="QHBoxLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QTabWidget" name="sshTabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="sshBasicTab">
            <attribute name="title">
             <string>&amp;Basic</string>
            </attribute>
            <layout class="QFormLayout">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <property name="labelAlignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <item row="2" column="0">
              <widget class="QLabel" name="label7">
               <property name="text">
                <string>&amp;Hostname:</string>
               </property>
               <property name="buddy">
                <cstring>editHostName</cstring>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="editHostName"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label8">
               <property name="text">
                <string>&amp;User:</string>
               </property>
               <property name="buddy">
                <cstring>editUserName</cstring>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="editUserName"/>
             </item>
             <item row="4" column="1">
              <layout class="QHBoxLayout">
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>70</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pushTestConnection">
                 <property name="text">
                  <string>Test C&amp;onnection...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="sshAdvanced">
            <attribute name="title">
             <string>&amp;Advanced</string>
            </attribute>
            <layout class="QFormLayout" name="formLayout2">
             <property name="labelAlignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label12">
               <property name="text">
                <string>SS&amp;H executable:</string>
               </property>
               <property name="buddy">
                <cstring>sshExecutableEdit</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="sshExecutableEdit"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label9">
               <property name="text">
                <string>SCP &amp;executable:</string>
               </property>
               <property name="buddy">
                <cstring>scpExecutableEdit</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="scpExecutableEdit"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label10">
               <property name="text">
                <string>&amp;Identity file (private key):</string>
               </property>
               <property name="buddy">
                <cstring>editIdentityFile</cstring>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLineEdit" name="editIdentityFile"/>
               </item>
               <item>
                <widget class="QToolButton" name="fileButton">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label11">
               <property name="text">
                <string>&amp;Port:</string>
               </property>
               <property name="buddy">
                <cstring>spinSshPort</cstring>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <layout class="QHBoxLayout" name="keyFileLayout">
               <item>
                <widget class="QSpinBox" name="spinSshPort"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="push_sleepTest">
          <property name="text">
           <string>Su&amp;bmit test job...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox4">
     <layout class="QVBoxLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Launcher template</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="1" column="2">
          <widget class="QPushButton" name="templateHelpButton">
           <property name="text">
            <string>Template &amp;help</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" colspan="3">
          <widget class="QPlainTextEdit" name="text_launchTemplate">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>400</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>text_launchTemplate</zorder>
        <zorder>templateHelpButton</zorder>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edit_submissionCommand</tabstop>
  <tabstop>edit_killCommand</tabstop>
  <tabstop>edit_requestQueueCommand</tabstop>
  <tabstop>updateIntervalSpin</tabstop>
  <tabstop>edit_launchScriptName</tabstop>
  <tabstop>edit_workingDirectoryBase</tabstop>
  <tabstop>wallTimeHours</tabstop>
  <tabstop>wallTimeMinutes</tabstop>
  <tabstop>sshTabWidget</tabstop>
  <tabstop>editHostName</tabstop>
  <tabstop>editUserName</tabstop>
  <tabstop>pushTestConnection</tabstop>
  <tabstop>sshExecutableEdit</tabstop>
  <tabstop>scpExecutableEdit</tabstop>
  <tabstop>editIdentityFile</tabstop>
  <tabstop>fileButton</tabstop>
  <tabstop>spinSshPort</tabstop>
  <tabstop>push_sleepTest</tabstop>
  <tabstop>text_launchTemplate</tabstop>
  <tabstop>templateHelpButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
